### ISSUE TITLE
TalkBack: Sign In: The social network links are announced without a name and link role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77468]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using TalkBack on Android, open the app
2. Swipe to focus the Phone or email text field and double-tap to activate it
3. Enter a registered email
4. Swipe to focus the Phone or email test field and double-tap to activate it
5. Swipe to focus the social icons at the footer and note how they are announced

## Expected Result:
The social network links/icons should be announced with a link role and descriptive name, e.g.:
Expensify: "Expensify, link. Double-tap to activate."
Twitter: "Twitter, link. Double-tap to activate."
Instagram: "Instagram, link. Double-tap to activate."
Facebook: "Facebook, link. Double-tap to activate."
LinkedIn: "LinkedIn, link. Double-tap to activate."

## Actual Result:
The social network links are announced without a name and link role.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Use the link role and descriptive accessibilityLabel to announce each icon link, e.g.:
- 'Twitter'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://appt.org/en/docs/react-native/samples/accessibility-link
https://appt.org/en/docs/react-native/samples/accessibility-role

## Environment:
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="1080" height="2195" alt="Image" src="https://github.com/user-attachments/assets/f1585a68-9462-437c-bc56-f01ede6e4e44" />

https://github.com/user-attachments/assets/ad582936-0161-4ffa-a7e5-23635b9f63cd

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7009028
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
dupe of 
- https://github.com/Expensify/App/issues/74512
